bite-size-notes/week-8-b-probability-review.md

```diff
@@ -32,13 +31,17 @@ To start our probability review chapter, we'll first define some important terms
 
```

Comment:
There are repeat symbols in the preview.

Reply:
Good catch, thank you!